bin/mirror-lens-references.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# This script assumes that `AWS_PROFILE` has been set to a AWS CLI
+# profile that has write-access to the `s3://sage-igenomes` bucket
+# (or using any other means of authenticating with the AWS CLI).
+
+# Downloading the LENS reference files will temporarily take up ~72.1 GB of disk space.
+
+prefixes=(
+    "antigen.garnish/"
+    "erv/"
+    "homo_sapiens/"
+    "mhcflurry/"
+    "mus_musculus/"
+    "viral/"
+)
+
+echo "Downloading LENS reference files"
+mkdir -p "./lens/references"
+cd ./lens
+wget -P . https://gitlab.com/landscape-of-effective-neoantigens-software/nextflow/modules/tools/lens/-/wikis/uploads/b0d8a24036628bd9e6c860eb531919c3/download_general_references.sh \
+    https://gitlab.com/landscape-of-effective-neoantigens-software/nextflow/modules/tools/lens/-/wikis/uploads/2238c7f4af5c5499f298315bf5080ad7/download_human_references.sh \
+    https://gitlab.com/landscape-of-effective-neoantigens-software/nextflow/modules/tools/lens/-/wikis/uploads/bf210753e3724edf6db2543eb54c3de4/download_mouse_references.sh
+export RAFT_PATH="."
```

Comment:
Also, did we want to include instructions for installing raft?

Reply:
You don't need to install RAFT to download the reference files so I don't think that's necessary here, but the process outlined in the LENS docs leads to you mounting these refs in your RAFT instance, hence the reference to `RAFT_PATH` here.

Reply:
Thanks, I'm loooking in the shell scripts, and it does seem like there are dependencies you needed to install like:

* singularity
* Python + certain python packages

Did you experience that?

Reply:
Yes, the instructions [here](https://gitlab.com/landscape-of-effective-neoantigens-software/nextflow/modules/tools/lens/-/wikis/Running-LENS#populate-raft-global-references-directory) include those installations, I will add this link to the script as set-up instructions